src/Processors/DataProcessor.php

```diff
@@ -98,21 +98,17 @@ public function process($object = false)
         $indexColumn  = config('datatables.index_column', 'DT_Row_Index');
 
         foreach ($this->results as $row) {
-            $data  = Helper::convertToArray($row);
-            $value = $this->addColumns($data, $row);
-            $value = $this->editColumns($value, $row);
-            $value = $this->setupRowVariables($value, $row);
-            $value = $this->selectOnlyNeededColumns($value);
-            $value = $this->removeExcessColumns($value);
-
-            if ($this->includeIndex) {
-                $value[$indexColumn] = ++$this->start;
-            }
-
+            $data           = $this->escapeRow(Helper::convertToArray($row));
+            $value          = $this->addColumns($data, $row);
+            $value          = $this->addIndexColumn($value, $row);
```

Comment:
I think this should be the last item in row processing or maybe second to the last then escape the row.

Reply:
We only escape row columns here cause we need to escape addon columns on their own function, so we can decide to not escape blade strings.